about.html

```diff
@@ -0,0 +1,159 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css">
+  <link rel="stylesheet" href="./styles/reset.css">
+  <link rel="stylesheet" href="./styles/about.css">
+</head>
+
+<body>
+  <header>
+    <section class="nave-header">
+      <div class="upper-nav">
+        <ul>
+          <li><i class="fa-brands fa-facebook-f"></i></li>
+          <li><i class="fa-brands fa-twitter"></i></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="desktop-menu">
+        <div id="desktop-menu__logo">
+          <a href="./index.html"> <img src="./assets/logo.png" alt="" srcset=""> </a>
+        </div>
+        <div class="desktop-menu__links">
+          <ul class="desktop-menu__links_list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="./index.html">CC Compaign</a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+    <nav role="navigation">
+      <div class="hamburger-menu">
+        <input id="menu__toggle" type="checkbox" />
+        <label class="menu__btn" for="menu__toggle">
+          <span></span>
+        </label>
+
+        <ul class="menu__box">
+          <li><a class="menu__item" href="./index.html">Home</a></li>
+          <li><a class="menu__item" href="./about.html">About</a></li>
+
+          <li><a class="menu__item" href="#">Contact</a></li>
+          <li><a class="menu__item" href="#">Twitter</a></li>
+        </ul>
+      </div>
+
+      <div class="about-div">
+
+        <h2 class="sub-header"> SOFTWARE DEVELOPER <br> GLOBAL SUMMIT 2023</h2>
+        <p class="paragraph"> Engage in insightful discussions, gain valuable knowledge, and connect with fellow
+          developers. This summit is your gateway to expanding your skills and embracing the future of software corporis
+          labore quidem architecto! Aut, voluptatibus esse molestiae similique excepturi consequuntur. Soluta beatae
+          explicabo aut ratione.
+          development.</p>
+        <p class="paragraph par"> Please contact us oer Email for any farther questions <br id="contact-email">
+          about software global summit
+        </p>
+        <h6> majdkilany92@gmail.com</h6>
+        <hr>
+      </div>
+
+      </div>
+
+  </header>
+  <main>
+    <section class="main-sec">
+
+      <h3 class="sub-header-main">
+        Creative Commons <br> Software Global Summit 2023
+        <hr class="hr">
+      </h3>
+      <p class="paragraph par">
+        The logo of Software Developer global summit was decided through the logo competition from 8.june to 7.july.
+
+      </p>
+
+      <div class="logo-container">
+        <img src="./assets/logo.png" alt="">
+
+      </div>
+
+    </section>
```

Comment:
Kindly update the background color of the middle section to be white. Please also add the divider at the bottom of the section

Your design:
<img width="494" alt="image" src="https://github.com/majdalkilany/Software-Developer-Summit/assets/37116322/3c07f09e-13d7-41aa-b00d-02af773ab92f">

Behance design:
<img width="556" alt="image" src="https://github.com/majdalkilany/Software-Developer-Summit/assets/37116322/06fbb46b-c257-464f-a6a3-cbd649ddc23a">